icpc-wf/ansible/roles/judgedaemon/tasks/main.yml

```diff
@@ -0,0 +1,87 @@
+---
+# These tasks configure the DOMjudge judgedaemon with chroot
+
+- name: create domjudge-run users
+  user: name={{item}} createhome=no home=/nonexistent group=nogroup shell=/bin/false
+  with_items:
+    - domjudge-run-0
+    - domjudge-run-1
+    - domjudge-run-2
+    - domjudge-run-3
+
+- name: create domjudge-run group
+  group: name=domjudge-run state=present
+
+- name: create temp package directory
+  file:
+    path: /tmp/dj_ansible
+    state: directory
+    owner: root
+    group: root
+    mode: 0700
+
+- name: copy chroot DEB packages to install
+  copy:
+    src: install-chroot
+    dest: /tmp/dj_ansible/
+
+- name: create chroot
+  shell: "{{DJ_DIR}}/misc-tools/dj_make_chroot -y -i openjdk-11-jdk-headless -l \"$(ls /tmp/dj_ansible/install-chroot/*.deb 2>/dev/null | tr '\n' ',')\" 2>&1 | tee /tmp/dj_make_chroot.log; grep '^Done building chroot in' /tmp/dj_make_chroot.log"
+  environment:
+    DEBMIRROR: "{{ 'https://packages/ubuntu' if WF_RESTRICTED_NETWORK else '' }}"
+    DEBPROXY: "http://router:3142"
```

Comment:
This should be a variable right?

Reply:
Oops, that `DEBPROXY` is my local testing stuff. Might make sense to introduce a variable or so to allow setting an APT proxy, but I'd leave that to a next PR.

Reply:
Fixed